Manifests/ManagedPreferencesApplications/com.1password.1password.plist

```diff
@@ -0,0 +1,391 @@
+			<key>pfm_description</key>
+			<string>If present enforces whether Apple Watch unlock is allowed (Preferences > Security > Unlock).</string>
+			<key>pfm_name</key>
+			<string>security.authenticatedUnlock.appleWatchUnlock</string>
+			<key>pfm_platforms</key>
+			<array>
+				<string>macOS</string>
+			</array>
+			<key>pfm_title</key>
+			<string>Allow Apple Watch Unlock</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
+		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>8.0</string>
+			<key>pfm_description</key>
+			<string>Enforces showing passwords and full credit card numbers (Preferences > Security > Conceal Fields).</string>
+			<key>pfm_documentation_url</key>
+			<string>https://support.1password.com/mobile-device-management/</string>
+			<key>pfm_name</key>
+			<string>security.revealPasswords</string>
+			<key>pfm_platforms</key>
+			<array>
+				<string>macOS</string>
+			</array>
+			<key>pfm_title</key>
+			<string>Allow revealing passwords</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
+		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>8.0</string>
+			<key>pfm_description</key>
+			<string>Enforces the configured locked on sleep,screensaver, or switching users (Preferences > Security > Auto-lock).</string>
+			<key>pfm_documentation_url</key>
+			<string>https://support.1password.com/mobile-device-management/</string>
+			<key>pfm_name</key>
+			<string>security.autolock.onDeviceLock</string>
+			<key>pfm_platforms</key>
+			<array>
+				<string>macOS</string>
+			</array>
+			<key>pfm_title</key>
+			<string>Lock on sleep, screensaver, or switching users</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
+		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>8.0</string>
+			<key>pfm_description</key>
+			<string>Enforces the configured lock on app exit preference</string>
+			<key>pfm_documentation_url</key>
+			<string>https://support.1password.com/mobile-device-management/</string>
+			<key>pfm_name</key>
+			<string>security.autolock.onWindowClose</string>
+			<key>pfm_platforms</key>
+			<array>
+				<string>macOS</string>
+			</array>
+			<key>pfm_title</key>
+			<string>Lock when main window is closed</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
+		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>8.0</string>
+			<key>pfm_description</key>
+			<string>Enforces the lock on idle time preference (Preferences > Security > Lock).</string>
+			<key>pfm_documentation_url</key>
+			<string>https://support.1password.com/mobile-device-management/</string>
+			<key>pfm_name</key>
+			<string>security.autolock.minutes</string>
+			<key>pfm_platforms</key>
+			<array>
+				<string>macOS</string>
+			</array>
+			<key>pfm_title</key>
+			<string>Set auto-lock timeout</string>
+			<key>pfm_type</key>
+			<string>integer</string>
+			<key>pfm_range_min</key>
+			<integer>1</integer>
+			<key>pfm_range_max</key>
+			<integer>1440</integer>
```

Comment:
nit: Would you mind ordering the keys here in the rest of this dictionary alphabetically?

Reply:
Fixed in [commit](https://github.com/ProfileCreator/ProfileManifests/pull/577/commits/e55f267622f1ac613b92f3560d17a842942edb2a)